package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nestjs-typegoose",
-  "version": "7.0.0",
+  "version": "7.1.0",
```

Comment:
i would recommend to revert this, the maintainer @kpfromer should only do this

Reply:
OK!

I was following the procedure outlined here: https://github.com/kpfromer/nestjs-typegoose/blob/master/CONTRIBUTING.md#pull-request-process

If PRs should not update version, perhaps that instruction should be removed from the contribution guidelines.

Reply:
OK nevermind, didnt read that, sorry - i thought it was "common knowledge" to not increment it, when not being able to release it (thanks to something like `npm version *` scripts)

Reply:
otherwise, to comply with semver:
because this is not a breaking feature, i would recommend only increment the patch number

Reply:
Hm, maybe that is from a different semver spec?

From https://semver.org/ , both v1.0.0 and v2.0.0 specs have a line starting with
> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API.

Reply:
ok, i thought about npm-semver https://docs.npmjs.com/about-semantic-versioning#incrementing-semantic-versions-in-published-packages

Reply:
nvm confused "Backward compatible ***bug fixes***" with "Backward compatible ***new features***"

Reply:
consider this resolved (cant resolve this)